### ISSUE TITLE
Could export UIBezierPath to SVG path?

### ISSUE THREAD

**Issue by arden** [timrwood/SVGPath#5]
Could support export UIBezierPath to SVG path?

**Comment by timrwood** [comment]
No, as far as I am aware, there is no way to introspect a `UIBezierPath` and turn it into an svg path.

It might be possible to create a `SVGPathBuilder` class, with the same interface as `UIBezierPath`, so that as long as you had the source code that created a `UIBezierPath`, you could change the constructor and generate the svg path string.
